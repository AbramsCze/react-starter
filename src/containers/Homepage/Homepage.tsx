// libs
import React, { Fragment, FunctionComponent } from 'react'
import { Link } from 'react-router-dom'

// others
import { LoadingButton } from '../../components'
import routes from '../../constants/routes'

export const Homepage: FunctionComponent = () => {
    return (
      <Fragment>
        <LoadingButton />
        <Link to={routes.USER_FORM}>Přidat uživatele</Link>
      </Fragment>
    )
}
