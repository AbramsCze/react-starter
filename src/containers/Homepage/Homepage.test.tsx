// libs
import React from 'react'
import { shallow } from 'enzyme'

// others
import { Homepage } from './Homepage'

describe('Homepage page', () => {
  it('should render', () => {
    const renderedApp = shallow(<Homepage />)
    expect(renderedApp).toMatchSnapshot()
  })
})
