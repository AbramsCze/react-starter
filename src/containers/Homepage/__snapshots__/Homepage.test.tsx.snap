// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage page should render 1`] = `ShallowWrapper {}`;
